src/db/postgres.ts

```diff
@@ -37,13 +37,15 @@ export class Postgres implements IDatabaseConnector {
 	constructor(private connectionString: string) {
 
 	}
-	public Open() {
+	public async Open() {
```

Comment:
you probably have to specify a return value here, so `public async Open(): Promise<whatever-the-type-is> {`

Reply:
Ah, will do, thanks for the note!

Reply:
```suggestion
	public async Open(): Promise<void> {
```